wordpress.yaml

```diff
@@ -1,7 +1,104 @@
 ---
 - name: Deploy Wordpress
   hosts: all
+  become: yes
   tasks:
-
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna be runing on the VM'
+    - name: Update & Upgrade the System
+      apt:
+        update_cache: yes
+        upgrade: yes    
+    
+    - name: Install LEMP Packages
+      apt:
+        name:
+            - nginx
+            - mysql-server
+            - php7.4-cli
+            - php7.4-fpm
+            - php7.4-mysql
+            - php7.4-json
+            - php7.4-opcache
+            - php7.4-mbstring
+            - php7.4-xml
+            - php7.4-gd
+            - php7.4-curl
+            
+    - name: Allow NGINX on Firewall   
+      ufw:
+        rule: allow
+        name: "Nginx Full"
+    
+    - name: Copy Nginx config       
+      copy:
+        src: /vagrant/wordpressCfg.txt
+        dest: /etc/nginx/sites-available
+        
+    - name: Symbolic link on Wordpress nginx configuration file      
+      file:
+        src: /etc/nginx/sites-available/wordpressCfg.txt
+        dest: /etc/nginx/sites-enabled/wordpressCfg.txt
+        state: link
+        
+    - name: Unlink Default nginx config      
+      file:
+        path: /etc/nginx/sites-enabled/default
+        state: absent
+        
+    - name: Create Wordpress Database   
```

Comment:
have you seen https://docs.ansible.com/ansible/latest/collections/community/mysql/mysql_db_module.html ?

Reply:
I found easier to install mysql database from an .sql configuration file. I could try that too.